bin/seeg-ct.sh

```diff
@@ -69,8 +69,8 @@ label_with_dilation("masked_CT.nii", "dilated_CT.nii", "labeled_CT.nii")
 EOF
 
 echo compare labels to implantation schema
-echo mrview T1.nii.gz -overlay.load=dilated_CT.nii
-echo $open $implantation_pptx
+echo mrview T1.nii.gz -overlay.load labeled_CT.nii -overlay.interpolation_off -overlay.colourmap 3
+echo open implantation_pptx
```

Comment:
I was thinking that some environment variables would do

- on mac, `open=open`
- on linux, `open=xdg-open`
-on windows (like we care, but for fun), `open=''`

but it's true that there isn't much we can do here